extensions/interactions/NumberWithUnits/directives/number-with-units-interaction.component.html

```diff
@@ -2,6 +2,7 @@
   <!-- This ng-form is needed for submitting answer on hitting enter. -->
   <input type="text"
          name="answer"
+         aria-label="Number with units"
```

Comment:
Not sure a user would understand this; what is the best practice for labelling input fields? Is it with an instruction or a description of what's typed there? (Perhaps point to a reference that supports what you decide to go with.)

Reply:
Labels need to describe the purpose of the form control. PTAL at this from [W3 guidelines](https://www.w3.org/WAI/tutorials/forms/labels/)

Reply:
Does the label you've chosen describe the *purpose* of the form control? It seems to be similar to labelling a text input box with "text'.

Reply:
Ahh! I got the point that you are trying to put, thanks for pointing it out. According to my understanding for this particular case it is best to give it the label of "Number with units". Ultimately the purpose of this input field will be conveyed to the user by the question that they might be answering. What do you think?

Reply:
Hi @seanlip I was checking some good sites to figure out what would be best practice here. I found khan academy using the aria label just as -- "Your answers:" in their website PTAL, I am not sure if that is fine. What do you think?

![Screenshot 2023-07-20 203924](https://github.com/oppia/oppia/assets/96219910/4cebb18b-8ebe-4bf0-a35b-19fc7e6e8376)

Reply:
"Your answer" seems reasonable to me (I suggest using the singular case since there's only one answer).

Reply:
@seanlip Done PTAL.